nemo_rl/data/datasets/response_datasets/conversation_base.py

```
@@ -0,0 +1,131 @@
+        return data
+
+
+def conversation_process_message(
+    metadata: Dict[str, Any],
+    message: Dict[str, str],
+    media_index: dict,
+    raw: Dict[str, Any] = {},
+    allow_empty_text: bool = False,
+    check_if_media_file_exist: bool = True,
+    tried_default_extensions: set = set(),
+    tags_mapping_sample_to_allowed: Dict[str, str] = multimodal_utils.media_tags_to_allowed,
```

Ruff (0.14.8)

Unused function argument: `tags_mapping_sample_to_allowed`

(ARG001)